front/src/modules/rollingStock/components/RollingStockSelector/SearchRollingStock.tsx

```diff
@@ -160,8 +174,15 @@ const SearchRollingStock = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [filters, rollingStocks]);
 
+  useEffect(() => {
+    if (mustResetFilters) {
+      resetFilters();
+      if (setMustResetFilters) setMustResetFilters(false);
```

Comment:
you can test this function in addition with mustResetFilters in the condition above and just call it here

Reply:
done